core/execution/future/market.go

```diff
@@ -963,6 +963,11 @@ func (m *Market) PostRestore(ctx context.Context) error {
 		}
 	}
 
+	// Disposal slippage was set as part of this upgrade, send event to ensure datanode is updated.
+	if vegacontext.InProgressUpgradeFrom(ctx, "v0.75.8") {
```

Comment:
Ran into this with the second bridge migrations:
We can't use `v0.75.8` because thats the special mainnet-only patch and stagnet/fairground will migration from `v0.75.7`. Which is annoying, so we'll have to just use `vgcontext.InProgressUpgrade(ctx)` and remember to whip it out asap once the release is on mainnet.

Reply:
we could just `InProgressUpgradeFrom(ctx, "v0.75.8") || InProgressUpgradeFrom(ctx, "v0.75.7")` no?

Reply:
seeing as we've had this happen twice (fairground <=> mainnet tags), I added `InProgressUpgradeFromMultiple`, and now check for both tags